evo-delivery-app/evo-delivery-backend/src/controllers/driver.controller.ts

```diff
@@ -1,39 +1,61 @@
 import { Request, Response, NextFunction } from 'express';
 import { driverService } from '../services';
+import { Driver } from '../types';
+import { INVALID, OK } from '../utils/response.utils';
 
-// Example for controller
-export const getDrivers = async (
-  req: Request,
-  res: Response,
-  next: NextFunction
-) => {
+export const getDrivers = async (req: Request, res: Response) => {
   const page = req.params.page || 1;
   const limit = req.params.limit || 10;
 
   try {
     const drivers = await driverService.getDrivers({}, page, limit);
-    return res.status(200).json({
-      status: 200,
-      data: drivers,
-      message: 'Successfully Drivers Retrieved',
-    });
+    return res.status(200).json(OK(drivers));
   } catch (e: any) {
-    return res.status(400).json({ status: 400, message: e.message });
+    return res.status(400).json(INVALID(400, e.message));
```

Comment:
Nice convention! Let's note to ourselves to add a logger and log each step with the appropriate log level (e.g 'error', 'warn' or 'info').

I think we should add 'application/json' header to the response (via middleware or something like this)